docs/source/contributing.rst

```diff
@@ -26,11 +26,11 @@ Installing a Development Version
 
       git clone --recurse-submodules git@github.com:<username>/scico.git
 
-4. Create a conda environment using Python 3.8 or later, e.g.:
+4. Create a conda environment using Python 3.10 or later, e.g.:
 
    ::
 
-      conda create -n scico python=3.9
+      conda create -n scico python=3.11
```

Comment:
I suggest `conda create -n temp "python>=3.10,<3.13"` as a more future-proof option.

Reply:
Or maybe `conda create -n scico python=3.10` if that really is the version that is most thoroughly tested.

Reply:
Switched to 3.12, which is the current stable version, and also the version tested in scico CI.